doc/developer/logging.rst

```diff
@@ -502,6 +502,45 @@ General utility formats
    representation for a hexdump.  Non-printable characters are replaced with
    a dot.
 
+.. frrfmt:: %pIN (struct iso_address *)
+
+   ([I]so [N]etwork address) - Format ISO Network Address
+
+   ``%pIN``: :frrfmtout:`01.0203.04O5`
```

Comment:
If we do rename the other into 3 distinct identifiers, I'd maybe pick %pIS for this one short for for "ISO Address" (It's really an OSI network address (ISO 8348), but this may be too far removed at this point to help anyone)

Reply:
Done